terraform/vars/terraform-dev.tfvars

```diff
@@ -4,3 +4,4 @@ tile_cache_url            = "dev-tiles.globalforestwatch.org"
 desired_count             = 1
 auto_scaling_min_capacity = 1
 auto_scaling_max_capacity = 5
+new_relic_license_key_arn      = "arn:aws:secretsmanager:us-east-1:563860007740:secret:newrelic/license_key-lolw24"
```

Comment:
Just curious.  We don't usually put arns directly in the terraform, right?  But I guess this is OK because ARNs for secrets in the secret manager do no change over the long term?  (I see that these arns are already in the terraform for the data API as well.)

Reply:
Yeah, the ARNs are not expected to change but I agree hard coding them isn't ideal. I have made a minor change in 325b17fcf9c3dcef446c6f692f63b89e0b456d8a where TF  fetches the secret arn from aws using the name attribute that's the same across environments and not editable.